packages/pangea-sdk/PangeaCyber.Net/Redact/Results/UnredactResult.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+
+namespace PangeaCyber.Net.Redact
+{
+    /// <kind>class</kind>
+    /// <summary>
+    /// RedactTextResult
+    /// </summary>
+    public sealed class UnredactResult
+    {
+        ///
+        [JsonProperty("data")]
+        public object data { get; private set; } = default!;
```

Comment:
Could this be made into a generic? Like `EncryptStructuredResult<T>`.

Reply:
Done